api/app/models/user.rb

```diff
@@ -2,36 +2,27 @@
 #
 # Table name: users
 #
-#  id                     :bigint(8)        not null, primary key
-#  academic_year          :integer(4)
-#  coop_cycle             :string
-#  email                  :string           default(""), not null
-#  encrypted_password     :string           default(""), not null
-#  graduation_year        :integer(4)
-#  image_url              :string
-#  is_advisor             :boolean          default(FALSE), not null
-#  major                  :string
-#  remember_created_at    :datetime
-#  reset_password_sent_at :datetime
-#  reset_password_token   :string
-#  username               :string
-#  created_at             :datetime         not null
-#  updated_at             :datetime         not null
+#  id              :bigint(8)        not null, primary key
+#  academic_year   :integer(4)
+#  coop_cycle      :string
+#  email           :string           default(""), not null
+#  graduation_year :integer(4)
+#  image_url       :string
+#  is_advisor      :boolean          default(FALSE), not null
+#  major           :string
+#  username        :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  nu_id           :string
 #
 # Indexes
 #
-#  index_users_on_email                 (email) UNIQUE
-#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
-#  index_users_on_username              (username)
+#  index_users_on_email  (email) UNIQUE
```

Comment:
Can we also index on username - Will be useful to get back list of sorted students.

Reply:
The issue with that is we set the username as first_name + last_name, and Ali said that are a bunch of duplicate names in Khoury.